module/behaviour/skills/WalkPathFollower/src/WalkPathFollower.cpp

```diff
@@ -223,22 +237,28 @@ namespace behaviour {
 
                             // emit(utility::nusight::drawArrow("WPF_Closest_Arrow",
                             // currentState.localToWorld(walkCommand->command), {1,1,1}, 1));
-                            arma::vec2 arrowTip = currentState.localToWorld(walkCommand->command).xy();
-                            arma::vec2 dirPoint =
-                                currentState
-                                    .localToWorld({walkCommand->command.rotation() * walkCommand->command.xy(), 0})
-                                    .xy();
-
-                            emit(utility::nusight::drawArrow("WPF_Closest_Arrow", currentState.xy(), arrowTip, 1));
-                            emit(utility::nusight::drawArrow("WPF_Closest_Arrow_Rotation", arrowTip, dirPoint, 1));
+                            Eigen::Affine2d command = Eigen::Affine2d::Identity();
+                            command.linear()        = Eigen::Rotation2Dd(walkCommand->command.z()).toRotationMatrix();
+                            command.translation()   = walkCommand->command.head<2>();
+                            Eigen::Affine2d walk    = Eigen::Affine2d::Identity();
+                            walk.linear()           = Eigen::Rotation2Dd(0.0).toRotationMatrix();
+                            walk.translation()      = Eigen::Rotation2Dd(walkCommand->command.z()).toRotationMatrix()
+                                                 * walkCommand->command.head<2>();
+
+                            // Eigen::Vector2d arrowTip = localToWorld(currentState, command).translation();
+                            // Eigen::Vector2d dirPoint = localToWorld(currentState, walk).translation();
+                            // emit(utility::nusight::drawArrow("WPF_Closest_Arrow", currentState.xy(), arrowTip,
+                            // 1)); emit(utility::nusight::drawArrow("WPF_Closest_Arrow_Rotation", arrowTip,
+                            // dirPoint, 1));
 
                             emit(std::move(walkCommand));
 
-                            emit(utility::nusight::drawRectangle("WPF_Closest",
-                                                                 RotatedRectangle(targetState, {0.12, 0.17}),
-                                                                 {0, 0, 0}));
+                            // emit(utility::nusight::drawRectangle(
+                            //     "WPF_Closest",
+                            //     RotatedRectangle<double>(targetState, Eigen::Vector2d(0.12, 0.17)),
+                            //     {0, 0, 0}));
```

Comment:
We used to have a heap of nusight helper functions for drawing certain shapes in nusight. These have all (or mostly) been removed now, probably because NUsight2 doesn't currently support this any more. I commented them out in case someone wants to fix this later.

Reply:
Probably just remove in that case. It'll come up again if someone needs it.

Reply:
I have removed all of the nusight `draw` emits